lib/fastlane/plugin/firebase_app_distribution/client/firebase_app_distribution_api_client.rb

```diff
@@ -105,7 +114,7 @@ def get_app(app_id, app_view = 'BASIC')
       #
       # Throws a user_error if the binary file does not exist
       def upload_binary(app_id, binary_path, platform)
-        connection.post(binary_upload_url(app_id), read_binary(binary_path)) do |request|
+        response = connection.post(binary_upload_url(app_id), read_binary(binary_path)) do |request|
```

Comment:
Should we be setting the internally visible `client_info` field here instead of the headers?

https://source.corp.google.com/piper///depot/google3/google/firebase/appdistro/v1/firebase_app_distribution.proto;l=216;rcl=389870755

Reply:
We decided to drop the `client_info` param and stick with headers. That was only introduced when we thought we couldn't set headers using the new ts_api_client call in fireconsole, but we're going back to making a vanilla POST call.